sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -130,7 +129,7 @@ public void setUp() throws ClassNotFoundException {
             cosmosTemplate = createCosmosTemplate(cosmosConfig, TestConstants.DB_NAME);
         }
 
-        collectionManager.ensureContainersCreatedAndEmpty(cosmosTemplate, Person.class,
+        collectionManager.ensureContainersCreatedAndEmpty(cosmosTemplate, Person.class, PersonCrossPartition.class,
```

Comment:
we shouldn't create the container here if `PersonCrossPartition` is not being used.